ExampleFhirClient/Readme.md

```diff
@@ -0,0 +1,25 @@
+# ExampleFhirClient
+
+Very Basic Simple C# command line program to search, read and post information to/from the CareEvolution FHIR server.
+
+## Authentication
+   
+This application is using a JWT token to request an OAuth 2.0 access token and client authentication.  The application uses the jwt.careevolution.com certificate and expects the OAuth client configruation (id: JWTClientCredentials).  The signing certificate is available here: https://fhir.careevolution.com/jwt.pfx with password == `jwt.careevolution.com`
+
+for more authentication information: https://github.com/HL7/bulk-data/blob/master/spec/authorization/index.md#obtaining-an-access-token
+
+## Basic usage
+
+Download the jwt.careevolution.com certificate and install locally.  In Visual Studio (or preferred editor/command line runner), open the ExampleFhirClient program and run it.  The output will display in the open dialog box.
```

Comment:
'install locally' needs more details I think - how to do it exactly

Do we specify anywhere which kind of system can be used? Windows version, Visual Studio...

Reply:
I've added more detail, specifying to import into local computer certificate store under _Trusted People_.